### PR TITLE
docs: merge contribute.md and development.md

### LINKED ISSUE #905: docs: merge contribute.md and development.md
These two documents seem to play the same role. And the contribute.md doesn't mentioned the development.md, but it is the default contributing guide linked in project README.

Goals:
1. Move "How to contribute" and "How to report a bug" to README.
2. Merge "How to add a new feature or change an existing one" with development.md
3. Rename development.md to contribute.md

Discussion is welcome.

### PR DESCRIPTION
**Pull request by bin-ya**
### What this PR does / why we need it:

Resolves  #905 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,3 +126,17 @@ We welcome all kinds of contributions from the open-source community, individual
...
+
+## How to contribute
+
+Most of the contributions that we receive are code contributions, but you can
+also contribute to the documentation or simply report solid bugs
+for us to fix.
+
+ For new contributors, please take a look at issues with a tag called [Good first issue](https://github.com/apache/apisix-ingress-controller/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [Help wanted](https://github.com/apache/apisix-ingress-controller/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+ ## How to report a bug
```
Comment (tao12345666333) [inline README.md:138]: Please remove spaces

**Review by tao12345666333 — COMMENTED** [review]
thanks

**Comment by tao12345666333** [comment]
Please check the CI errors. 
You can run `make update-mdlint` locally.

**Comment by bin-ya** [comment]
> 请检查 CI 错误。 您可以在`make update-mdlint`本地运行。

development.md after renaming, there are three files containing development.md. Can I modify it?(README.md line 92, CHANGELOG.md line 419, getting-started.md line 70)

**Comment by tao12345666333** [comment]
> development.md after renaming, there are three files containing development.md. Can I modify it?(README.md line 92, CHANGELOG.md line 419, getting-started.md line 70)

Sure, please!

File: docs/en/latest/contribute.md [hunk 2]
```diff
@@ -21,23 +21,107 @@ title: Contributing to apisix-ingress-controller
...
+make update-mdlint
+```
+
+or just run one command:
+
+```shell
+make update-all
+```
 
 ## How to add a new feature or change an existing one
```
Comment (lingsamuel) [inline docs/en/latest/contribute.md:126]: I think it would be better to put this section before `test`.

Comment (bin-ya) [reply docs/en/latest/contribute.md:126]: Line 122 to 124 or the paragraph Pre-commit todo?

Comment (lingsamuel) [reply docs/en/latest/contribute.md:126]: I mean the "How to add a new feature or change an existing one" section, sorry.

**Review by lingsamuel — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Comment by lijing-21** [comment]
Hi @bin-ya  , thank you for your contribution!

Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)

[1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Pull request merged**